helpers/sql-processor/lib/opentelemetry/helpers/sql_processor/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Helpers
+    module SqlProcessor
+      VERSION = '0.1.0'
```

Comment:
@hannahramadan this is the first time you are creating this gem right? 

I believe needs to start with 0.0.0 and then the release process using toys will handle versioning.